src/stdune/csexp.mli

```diff
@@ -1,12 +1,15 @@
 type t = Sexp.t
 
+(** [to_string sexp] converts S-expression [sexp] to a string in canonical
+  form. *)
+val to_string : t -> string
+
+(** [of_string string] parses [string] into S-expression *)
+val of_string : string -> (t, string) Result.t
```

Comment:
If it's doing parsing, then it should just be called `parse` or `parse_string`. I recall we had this discussion regarding `Path` before (see Path_intf)